testing/trino-tests/src/test/resources/sql/presto/tpcds/iceberg/parquet/unpartitioned/q09.plan.txt

```diff
@@ -12,6 +12,11 @@ cross join:
                                             cross join:
                                                 cross join:
                                                     cross join:
+                                                        final aggregation over ()
```

Comment:
@findepi @raunaqmorarka is there a good way to tell if these changes are good or not?

Reply:
Sure, just run the benchmarks :)

Reply:
@alexjo2144 do you have the results?